src/components/toolbox/tabs/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import styles from './tabs.css';
+
+const Tabs = ({
+  tabs = [], active, onClick = () => {}, className,
```

Comment:
IMO, it's better written with the `defaultProps`, since here you could have just the names of props.

Reply:
Done